### ISSUE TITLE
[Guess] Looks like we have CRLF sneaked to the source code tree

### ISSUE THREAD

**Issue by USSX-Hares** [lidatong/dataclasses-json#461]
### Description

I was browsing the project documentation and noticed that all source code snippets have an extra newline after each normal line.
This is usually caused by some software processing CRLF as two separate and independent newline symbols.

### Code snippet that reproduces the issue

https://lidatong.github.io/dataclasses-json/reference/dataclasses_json/api/

![image](https://github.com/lidatong/dataclasses-json/assets/29100587/86157228-2253-4a60-a7da-27eca98a0e44)

### Describe the results you expected

n/a

### Python version you are using

n/a

### Environment description

n/a

**Comment by george-zubrienko** [comment]
Yeah windows users leaking the poison into the source :) Unfortunately, we can't update the line-endings only since git will ignore that, but we can add `.gitattributes` file that forces commits to be `LF`?

**Comment by USSX-Hares** [comment]
> Yeah windows users leaking the poison into the source :) Unfortunately, we can't update the line-endings only since git will ignore that, but we can add `.gitattributes` file that forces commits to be `LF`?

Actually, committing the EoL changes only is possible, but that drastically garbages the Git history.
I've done that kind of purification a few times for my own repos.

**Comment by USSX-Hares** [comment]
As a workaround for the documentation page, I can suggest adding `core.autocrlf = input` option to the GitHub Workflows checkout config.

**Comment by george-zubrienko** [comment]
Will send a PR later this week
